### ISSUE TITLE
Start server at specific time/date

### ISSUE THREAD

**Issue by Kugelschieber** [assetto-corsa-web/accweb#88]

**Comment by Menteroth** [comment]
Maybe add scheduled restart of server(s) as well? If possible.

**Comment by Kugelschieber** [comment]
You mean automatically restarting them at a certain time? What would that be good for?

**Comment by Menteroth** [comment]
Yeah that. Because long endurance races seems to take a toll on the server and create issues. That's what it sounded like at least.  Not entirely sure. So it sounded like it could be a decent enough feature perhaps.

I'll see if I can ask around too.

**Comment by Menteroth** [comment]
Asked around a bit. Didn't seem to be a problem.

**Comment by arelstone** [comment]
Tjis would be very usefull for us at Light Speed Racing. We are comming from OGP and was using the cron module to automatic start race servers when they were suppose to go live

**Comment by Renegade89** [comment]
For those interested in scheduled restart;

I found that deleting the cfg folder after starting the server will still run a full cycle of the configured sessions (for example P/Q/R, Q/R). After sessions have run it will output results files nicely, and the server will halt because of not having a valid config file available.

Using task scheduler you could create a batch script that runs the accserver.exe (or call API) and then delete the (contents of the) cfg-folder a few seconds later. This way only 1 practice, qualifying and race session will run. Ideal for league circumstances.
